web-lib-funcs.pl

```diff
@@ -5639,19 +5639,19 @@ sub get_module_overrides
 my $mdir = &module_root_directory($mod);
 
 # Call module specific overrides
-if (-r "$mdir/module.overrides" && !$main::get_module_overrides_done++) {
+if (-r "$mdir/module.overrides") {
 	my %overrides;
 	&read_file_cached("$mdir/module.overrides", \%overrides);
 	my $funcs = $overrides{'funcs'};
 	if ($funcs) {
 		eval {
 			local $main::error_must_die = 1;
 			my @funcs = split(/\s+/, $funcs);
-			&foreign_require($mod);
+			&foreign_require($mod, "$mod-lib.pl");
```

Comment:
This is an important fix as we need to define module's lib manually as `foreign_require` will call `get_module_info` in auto detection mode causing endless loop.

We should assume any module's overrides sub reside in main module's lib file.